index.html

```diff
@@ -2,14 +2,29 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="description" content="lesson-9">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Natalia's page</title>
 </head>
 <body>
+
+
+  <nav>
+    <a href="/css/">About</a> |
+    <a href="/js/">Portfolio</a> |
+    <a href="/python/">Blog</a>
+    <a href="/html/">Contact</a> |
+  </nav>
```

Comment:
That's awesome that you put a `nav` in your page, Natalia! However, I don't think it works the way you expect it to, right now. 

If you want those links to send the user to specific parts of this page, you need to pass the anchors to the existing sections you want each link to send to. Like this:

```suggestion
  <nav>
    <a href="#about">About</a> |
    <a href="#experience">Experience</a> |
    <a href="#connect">Connect</a>
  </nav>
```

Or, if those are supposed to use external links, you can pass those links as the `href` attribute for each link element:

```suggestion
  <nav>
    <a href="https://linktomyportfolio.com">Portfolio</a> |
    <a href="https://linktomyblog.com">Blog</a>
    ...
  </nav>
```